bespin-web/scripts/start-apache.sh

```diff
@@ -6,7 +6,6 @@ python manage.py migrate
 create-lando-user.sh
 create-lando-connection.sh
 create-group-manager-connection.sh
```

Comment:
While we're cleaning this up, let's move the rest of the one-time setup out of the `start-apache.sh` script. The contents of these scripts could go into a single one-time configuration script

Reply:
How do you feel about me removing all these scripts and putting them in ansible?
They are all basically `python manage.py <command> <env/path_from_ansible>`.

Reply:
👍